### DISCUSSION TITLE
is there a way to put custom javascript on my post?

### DISCUSSION THREAD

**Discussion by danish-mehmood** [dillonzq/LoveIt#877] [Q&A]
how to put the javascript on my post page ?
consider the scenario where i want to animate an image i have put on my page , how would i put the javascript on the page to animate the image?

**Comment by XinuSys** [comment]
Please explain what you wish to achieve as animation. Do you have images you want to loop together? Do you have a reel to use as slideshow? Or is it simple scaling, rotation and translation of an element?
